### PR TITLE
Feat/40 Styles for transaction and transaction details page

### LINKED ISSUE #40: tx & tx/:id styles

### PR DESCRIPTION
**Pull request by sam-keen**
Closes #40 

Styling for txs and txs:id routes

Screenshot 2022-03-16 at 15 41 31

Screenshot 2022-03-16 at 15 41 42

### CODE REVIEW

File: apps/explorer/src/app/components/txs/id/tx-content.tsx [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
-
-  const displayCode = {
-    market: marketId,
-    type,
-    side,
-    size,
-  };
-
   return (
     <>
```
Comment (dexturr) [inline apps/explorer/src/app/components/txs/id/tx-content.tsx:19]: Unessecary fragment

File: libs/ui-toolkit/src/components/lozenge/lozenge.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  variant?: 'success' | 'warning' | 'danger' | 'highlight';
+  className?: string;
+  details?: string;
+}
+
+const getWrapperClasses = (className: LozengeProps['className']) => {
+  return classNames('inline-flex items-center gap-4', className);
+};
+
+const getLozengeClasses = (variant: LozengeProps['variant']) => {
```
Comment (dexturr) [inline libs/ui-toolkit/src/components/lozenge/lozenge.tsx:15]: Is this duplicated somewhere else in the ui toolkit? Can we have a helper function that takes intent and spits out background/text color classes if so

File: apps/explorer/src/app/components/render-fetched/index.tsx [hunk 3]
```diff
@@ -19,7 +19,9 @@ export const RenderFetched = ({
   }
```
Comment (dexturr) [inline apps/explorer/src/app/components/render-fetched/index.tsx:19]: Sorry we really should have done this earlier and I apologise, but we need some unit tests for the completed components, can you add tests for the ones in this PR please and thank you 🙏

**Review by mattrussell36 — COMMENTED** [review]
Looks pretty good to me aside from Dex's comments.

File: apps/explorer/src/app/components/txs/id/tx-content.tsx (hunk 1 above)
Comment (sam-keen) [reply apps/explorer/src/app/components/txs/id/tx-content.tsx:19]: Needed a parent fragment, but have changed the structure a little as I don't need the conditional rendering that follows the table.

File: libs/ui-toolkit/src/components/lozenge/lozenge.tsx (hunk 2 above)
Comment (sam-keen) [reply libs/ui-toolkit/src/components/lozenge/lozenge.tsx:15]: It's not yet as far as I've seen. I'll keep an eye out and make one if we get a second instance of this though.

File: apps/explorer/src/app/components/seconds-ago/index.tsx [hunk 4]
```diff
@@ -18,10 +18,16 @@ export const SecondsAgo = ({ date }: SecondsAgoProps) => {
     return <>Date unknown</>;
   }
 
+  console.log(
+    `now: ${now}, before: ${new Date(
+      date
+    ).getTime()}, date getting passed in: ${date}`
+  );
```
Comment (dexturr) [inline apps/explorer/src/app/components/seconds-ago/index.tsx:25]:
```suggestion
```

File: apps/explorer/src/app/components/seconds-ago/seconds-ago.spec.tsx [hunk 5]
```diff
@@ -1,24 +1,37 @@
...
+    const secondsToWait = 10;
     const dateInString = new Date().toString();
 
-    await new Promise((r) => setTimeout(r, secondsToWait * 1000));
+    act(() => {
+      jest.advanceTimersByTime(secondsToWait * 1000);
+    });
+
+    jest.runOnlyPendingTimers();
+    done();
```
Comment (dexturr) [inline apps/explorer/src/app/components/seconds-ago/seconds-ago.spec.tsx:29]:
```suggestion
```

File: apps/explorer/src/app/components/seconds-ago/seconds-ago.spec.tsx [hunk 6]
```diff
@@ -1,24 +1,37 @@
...
-    expect(baseElement).toBeTruthy();
+    const dateInString = new Date().toString();
+    render(<SecondsAgo data-testid="test-seconds-ago" date={dateInString} />);
+
+    expect(screen.getByTestId('test-seconds-ago')).toBeInTheDocument();
   });
 
-  it('should show the correct amount of seconds ago', async () => {
-    const secondsToWait = 2;
+  it('should show the correct amount of seconds ago', (done) => {
```
Comment (dexturr) [inline apps/explorer/src/app/components/seconds-ago/seconds-ago.spec.tsx:20]:
```suggestion
  it('should show the correct amount of seconds ago', () => {
```

File: apps/explorer/src/app/components/seconds-ago/index.tsx (hunk 4 above)
Comment (sam-keen) [reply apps/explorer/src/app/components/seconds-ago/index.tsx:25]: Whoops! Thanks!

**Review by dexturr — APPROVED** [review]

**Pull request merged**